README.md

```diff
@@ -9,23 +9,48 @@ This page is a tourist guide to the beautiful city of Lisbon. It's on GitHub bec
 
 * [The Nations Park Gondola Lift][3] (Cable Car)
 * [São Jorge Castle][4]
-* [Elevador da Bica)[5]
+* [Elevador da Bica][5]
```

Comment:
why not link it straight away? Footnotes are lovely and all, but in this context?